ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -110,6 +110,16 @@ class Tab: NSObject {
 
   private var observers: Set<AnyTabObserver> = .init()
 
+  // WebKit handlers
+  private var navigationHandler: TabWKNavigationHandler?
+  private var uiHandler: TabWKUIHandler?
+
+  var certStore: CertStore?
```

Comment:
Does `certStore` need to be weak? I don't think so but it might since `BrowserViewController` owns the `profile` that owns the `certStore`.

Reply:
Verified no tab leak